agent/php_wrapper.c

```diff
@@ -73,6 +73,26 @@ nruserfn_t* nr_php_wrap_user_function_before_after_clean(
   return wraprec;
 }
 
+nruserfn_t* nr_php_wrap_user_function_before_after_clean_extra(
```

Comment:
I'm sure it could be, but given that we're essentially just wrapping existing functionality of `nr_php_wrap_user_function_before_after_clean` I wouldn't consider it actually testing much and probably not worth the effort. If you look at the existing tests for `nr_php_wrap_user_function_before_after_clean` in `test_php_wrapper.c` you can see how involved tests involving this family of functions are.

Reply:
After f3164c6d31bcd5a4ea17d20d6c4d43983ed3018d I'm ok with not adding unit tests for this function.